_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
+import { hashHistory } from 'react-router';
```

Comment:
What should the order be? Seems like alphabetical in most places, but here not sure if there was another order on top of alphabetical.